astroid/nodes/as_string.py

```diff
@@ -156,7 +168,11 @@ def visit_binop(self, node):
     def visit_boolop(self, node):
         """return an astroid.BoolOp node as string"""
         values = [f"{self._precedence_parens(node, n)}" for n in node.values]
-        return (f" {node.op} ").join(values)
+        # return (f" {node.op} ").join(values)
+        tmp = (f" {node.op} ").join(values)
+        tmp = f"({tmp})"
+        # print("({})".format(tmp))
+        return tmp
```

Comment:
What is the reason for this change?

Reply:
Thanks for your reply.
When I test the third-party library "sklearn"(scikit-learn, https://github.com/scikit-learn/scikit-learn/tree/main/sklearn), I encountered these problems and modifications. 
You can try to transfer the "sklearn" library, and then execute.